primitives/test-utils/src/lib.rs

```diff
@@ -132,20 +179,25 @@ pub fn voter_set() -> VoterSet<AuthorityId> {
 
 /// Convenience function to get a list of Grandpa authorities.
 pub fn authority_list() -> AuthorityList {
+	keyring().iter().map(|(id, w)| (id.into(), *w)).collect()
+}
+
+/// Get the corresponding identities from the keyring for the "standard" authority set.
+pub fn keyring() -> Vec<(Keyring, u64)> {
 	vec![(alice(), 1), (bob(), 1), (charlie(), 1)]
 }
 
-/// Get the Public key of the Alice test account.
-pub fn alice() -> AuthorityId {
-	Ed25519Keyring::Alice.public().into()
+/// Convenience function to get a handle to the Alice test account.
+pub fn alice() -> Keyring {
+	Keyring::Alice
 }
 
-/// Get the Public key of the Bob test account.
-pub fn bob() -> AuthorityId {
-	Ed25519Keyring::Bob.public().into()
+/// Convenience function to get a handle to the Bob test account.
+pub fn bob() -> Keyring {
+	Keyring::Bob
 }
 
-/// Get the Public key of the Charlie test account.
-pub fn charlie() -> AuthorityId {
-	Ed25519Keyring::Charlie.public().into()
+/// Convenience function to get a handle to the Charlie test account.
+pub fn charlie() -> Keyring {
+	Keyring::Charlie
 }
```

Comment:
Is it that much of a convenience? You can actually do `use Keyring::*` and simply use `Alice` instead of `alice()`.

Reply:
I kept it more to reduce the changes in the existing code, since `alice()` and friends were already being used. Using the variants directly doesn't sound bad though